pyscriptjs/src/components/base.ts

```diff
@@ -120,30 +111,29 @@ export class BaseEvalElement extends HTMLElement {
         console.log('evaluate');
         this.preEvaluate();
 
-        const pyodide = runtime;
         let source: string;
-        let output;
+        let output: string;
         try {
             source = this.source ? await this.getSourceFromFile(this.source)
                                  : this.getSourceFromElement();
             const is_async = source.includes('asyncio')
 
-            await this._register_esm(pyodide);
+            await this._register_esm(runtime);
             if (is_async) {
-                await pyodide.runPythonAsync(
+                <string>await runtime.runPythonAsync(
                     `output_manager.change(out="${this.outputElement.id}", err="${this.errorElement.id}", append=${this.appendOutput ? 'True' : 'False'})`,
                 );
-                output = await pyodide.runPythonAsync(source);
+                output = <string>await runtime.runPythonAsync(source);
             } else {
-                output = pyodide.runPython(
+                output = <string>runtime.runPython(
```

Comment:
```suggestion
                output = <string>runtime.runPython(
```

Can't `runPython` return anything?

Reply:
looking at [their docs](https://pyodide.org/en/stable/usage/api/js-api.html#pyodide.runPython ), it returns `any`